package.json

```diff
@@ -361,6 +373,7 @@
         "js-yaml": "^3.13.1",
         "jsonc-parser": "^2.0.2",
         "lodash": "^4.17.11",
+        "marked": "^0.6.2",
```

Comment:
Does `marked` belong in dependencies or devDependencies?

Reply:
Haven't tried devDependencies, I'll see if that works.

Reply:
Looks like moving + `npm clean` + `npm install` + `npm run recompile` works. Leaving it in `devDependencies`.

Reply:
✔️